it/mail/chrome/messenger/offline.properties

```diff
@@ -1,20 +1,20 @@
 # Download Messages Prompt
-downloadMessagesWindowTitle1=Ricezione messaggi
-downloadMessagesLabel1=Scaricare i messaggi da usare una volta non in linea prima di scollegarsi?
-downloadMessagesCheckboxLabel1=Chiedi sempre prima di passare non in linea
+downloadMessagesWindowTitle1=Download messaggi
+downloadMessagesLabel1=Vuoi scaricare i messaggi da usare una volta non in linea prima di scollegarti?
+downloadMessagesCheckboxLabel1=Chiedi sempre quando passo non in linea
 
 # Send Messages Prompt
 sendMessagesWindowTitle1=Messaggi non inviati
-sendMessagesLabel1=Do you want to send the messages in your Unsent Messages folder?
-sendMessagesCheckboxLabel1=Chiedi sempre al momento di collegarsi.
+sendMessagesLabel1=Vuoi inviare i messaggi nella tua cartella messaggi non inviati?
+sendMessagesCheckboxLabel1=Chiedi sempre quando mi collego.
 
-# GetMessages While Offline Prompt 
-getMessagesOfflineWindowTitle1=Scarica messaggi 
-getMessagesOfflineLabel1=Al momento si è non in linea. Collegarsi per scaricare i nuovi messaggi?
+# GetMessages While Offline Prompt
+getMessagesOfflineWindowTitle1=Scarica messaggi
+getMessagesOfflineLabel1=Al momento sei non in linea. Vuoi collegarti per scaricare i nuovi messaggi?
 
 # Send Messages Offline Prompt
 sendMessagesOfflineWindowTitle1=Messaggi non inviati
-sendMessagesOfflineLabel=You are currently offline. Do you want to go online to send unsent messages?
+sendMessagesOfflineLabel=Al momento sei non in linea. Vuoi collegarti per inviare i messaggi non inviati?
```

Comment:
CORRECTION NEEDED. 

Replace "non in linea" with "offline" and "collegarti" with "passare online".